packages/builder/src/components/common/DrawerBindableInput.svelte

```diff
@@ -49,11 +50,13 @@
     <Button thin blue on:click={handleClose}>Save</Button>
   </heading>
   <div slot="body">
-    <BindingPanel
+    <svelte:component
+      this={panel}
       value={readableValue}
       close={handleClose}
       on:update={event => (tempValue = event.detail)}
-      bindableProperties={bindings} />
+      bindableProperties={bindings}
+      {bindings} />
```

Comment:
Probably shouldn't pass the same value in as 2 different props, maybe just update the automation drawer to use the `bindableProperties` prop instead? Or update `BindingPanel` to use `bindings` rather than `bindableProperties`. I don't mind either way, but we should probably just use one name.

Reply:
Makes sense to me. Should all be resolved now